config.py

```diff
@@ -2,12 +2,19 @@
 Global Configuration for Application
 """
 import os
+import json
 
 # Get configuration from environment
 DATABASE_URI = os.getenv(
-    "DATABASE_URI", "postgres://postgres:postgres@localhost:5432/postgres"
+    "DATABASE_URI", 
+    "postgres://postgres:postgres@localhost:5432/postgres"
 )
 
+# override if we are running in Cloud Foundry
+if 'VCAP_SERVICES' in os.environ:
+ vcap = json.loads(os.environ['VCAP_SERVICES'])
+ DATABASE_URI = vcap['user-provided'][0]['credentials']['url']
```

Comment:
this looks good, but we just need to make 15 and 16 indented I think

Reply:
> this looks good, but we just need to make sure lines 15 and 16 are indented

Oh my bad. Thank you.